src/Sylius/Behat/Context/Transform/ProductVariantContext.php

```diff
@@ -41,6 +42,22 @@ public function __construct(ProductRepositoryInterface $productRepository, Produ
     }
 
     /**
+     * @Transform :variant
+     * @Transform /^in variant "["^]+"$/
```

Comment:
We should always use already existing transformer `([^"]+)" variant of product "([^"]+)"`, when do we need variant without the context of which product?